Dockerfile.tmpl

```diff
@@ -172,7 +163,9 @@ RUN pip install spacy && \
 {{ if eq .Accelerator "gpu" }}
 # Install GPU-only packages
 # No specific package for nnabla-ext-cuda 11.x minor versions.
-RUN pip install pycuda \
+RUN export PATH=/usr/local/cuda/bin:$PATH && \
```

Comment:
@djherbis @psbang Any reasons why you updated the PATH and CUDA_ROOT? Also, a comment would be helpful to explain to future readers.

Reply:
I read it as a solution in a stackoverflow somewhere, I'll make sure to add the links in next time.

Reply:
A solution for what? You are removing the package in the end no?

Reply:
I was getting "Failed building wheel for pycude" errors, I found a solution here: https://forums.developer.nvidia.com/t/cant-install-pycuda/238230